### PR TITLE
CRM-20987, added batch transaction column

### PR DESCRIPTION
**Pull request by pradpnayak**
----------------------------------------
* CRM-20987: Add transaction date field to listings of transactions
  https://issues.civicrm.org/jira/browse/CRM-20987

Overview
----------------------------------------
Financial batch listing page lists transaction with transaction date derived from civicrm_financial_trxn table(column name = Received). This PR fixes 
- Renaming  column name from Received to Transaction Date
- Adding column Received derived from civicrm_contribution.receive_date

Before
----------------------------------------
before

After
----------------------------------------
batch transaction listing

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test failure looks like code just needs adjusting

File: CRM/Financial/Page/AJAX.php [hunk 1]
```diff
@@ -255,6 +255,7 @@ public static function getFinancialTransactionsList() {
       3 => 'amount',
       4 => 'trxn_id',
       5 => 'transaction_date',
+      5 => 'receive_date',
```
Comment (eileenmcnaughton) [inline CRM/Financial/Page/AJAX.php]: You have double indexed here. Might need to renumber?

**Comment by eileenmcnaughton** [comment]
The changed function is only accessed from the BatchTransction.tpl file - which relates to financial batches. I haven't tested this yet (maybe @lcdservices has) but the code side of it appears safe & ok except for my comment about array indexing

**Comment by pradpnayak** [comment]
@eileenmcnaughton  i have fixed the indexing.

**Comment by lcdservices** [comment]
tested and confirmed. looks good.

**Comment by eileenmcnaughton** [comment]
Adding merge on pass based on Brian's feedback

**Comment by pradpnayak** [comment]
jenkins restest this please

**Comment by pradpnayak** [comment]
jenkins, retest this please

**Comment by monishdeb** [comment]
Merging as per tag

**Pull request merged**